std/experimental/ndslice/slice.d

```diff
@@ -611,6 +611,43 @@ pure nothrow unittest
 }
 
 /++
+Creates an untinitilized array and an n-dimensional slice over it.
+Params:
+    lengths = list of lengths for each dimension
+    slice = slice to copy shape and data from
+Returns:
+    untinitilized n-dimensional slice
++/
+Slice!(Lengths.length, Select!(replaceArrayWithPointer, T*, T[]))
+untinitilizedSlice(T,
+    Flag!`replaceArrayWithPointer` replaceArrayWithPointer = Yes.replaceArrayWithPointer,
```

Comment:
Just use `Flag!"replaceArrayWithPointer" f = Yes.replaceArrayWithPointer`. After using `Flag` for a while I figured the name of the flag itself does not matter at all; its meaning is already present in the type and the constants involved.